js/core.js

```diff
@@ -1371,6 +1372,43 @@ var ERMrest = (function (module) {
             return (column.table.schema.name === this.table.schema.name &&
             column.table.name === this.table.name &&
             column.name === this.name);
+        },
+
+        // find the null value for the column based on context and annotation
+        _getNullValue: function (context) {
+            if (context in this._nullValue) { // use the cached value
+                return this._nullValue[context];
+            }
+
+            var value = -1,
+                displayAnnot = module._annotations.DISPLAY,
```

Comment:
Gah sorry I knew I was missing something. You're right; this is fine. I just missed the comma at the end of line 1383.